aten/src/ATen/native_parse.py

```diff
@@ -4,35 +4,126 @@ def python_num(s):
     except Exception:
         return float(s)
 
+CPP_TO_ATEN_TYPE_MAP = {
+  'std::vector<Tensor>': 'TensorList',
+}
+
+# change C++ types into ATen types (for output to Declarations.yaml);
+# e.g. std::vector<Tensor> -> TensorList, const/reference modifiers on Tensor
+# go away and are eventually restored to dynamic_type (this can probably be simplified
+# to a single transformation).
```

Comment:
Do we do any of this kind of parsing anywhere else in ATen, or just here?

Reply:
nn_parse does it but that's parsing nn headers that have TH types.